project_proposal/tex/mainpage.tex

```diff
@@ -108,45 +108,56 @@ \subsection{Summary of requirements}
 The system must be able to find the optimal production speed for each product type, based on an error simulation and the appertaining graph upon which the error simulation is built.
 
 \subsection{List of requirements}
+Below is a list of the above requirements. These requirements have been
+prioritised using the MoSCoW method, where M is for Must have, S is for
+Should have, C is for could have, and W is for Won't have. 
+
 \begin{table}[H]
-    \begin{tabularx}{\textwidth}{|>{\RaggedRight}p{1cm}|>{\RaggedRight}p{4cm}|>{\RaggedRight}X|}
+    \begin{tabularx}{\textwidth}{|>{\RaggedRight}p{1cm}|>{\RaggedRight}p{4cm}|>{\RaggedRight}X|>{\RaggedRight}p{1cm}}
+        \hline
+        \textbf{ID} & \textbf{Name} & \textbf{Description} & M \\
```

Comment:
should this line be like this? 

or like this? 
![image](https://user-images.githubusercontent.com/55736468/95689766-4b2bb680-0c13-11eb-827f-391b13e35472.png)

Reply:
hey this still says M instead of Priority 
Is it supposed to say that?